easybuild/easyconfigs/p/PyTorch/PyTorch-0.3.1-intelcuda-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,107 @@
+        'filename': 'gloo-20180105.tar.gz',
+        'extract_cmd': extract_cmd_pattern % (pytorchdir, 'torch/lib/gloo'),
+    },
+    {
+        'download_filename': 'ec44c6c1675c25b9827aacd08c02433cccde7780.tar.gz',
+        'filename': 'googletest-20160714.tar.gz',
+        'extract_cmd': extract_cmd_pattern % (pytorchdir, 'torch/lib/gloo/third-party/googletest'),
+    },
+    {
+        'download_filename': '14efb1a47a496652ab08b1ebcefb0ea24ae4a5e4.tar.gz',
+        'filename': 'nanopb-20170723.tar.gz',
+        'extract_cmd': extract_cmd_pattern % (pytorchdir, 'torch/lib/nanopb'),
+    },
+    {
+        'download_filename': '9f6a636e547fc70a02fa48436449aad67080698f.tar.gz',
+        'filename': 'pybind11-20170821.tar.gz',
+        'extract_cmd': extract_cmd_pattern % (pytorchdir, 'torch/lib/pybind11'),
+    },
+    {
+        'download_filename': '6a00cbc4a9b8e68b71caf7f774b3f9c753ae84d5.tar.gz',
+        'filename': 'clang-cindex-python3-20170330.tar.gz',
+        'extract_cmd': extract_cmd_pattern % (pytorchdir, 'torch/lib/pybind11/tools/clang'),
+    },
+]
+patches = [
+    '%(name)s-%(version)s-fix-for-intel.patch',
+    '%(name)s-%(version)s-cmake.patch',
+    '%(name)s-%(version)s_skip_MPI_tests.patch',
+    '%(name)s-%(version)s_intel_skip_autgrad_test.patch',
+    '%(name)s-%(version)s_fix_findAVX.patch',
+    '%(name)s-%(version)s_Fix_pushback_type.patch',
+    '%(name)s-%(version)s_Fix_static_assert.patch'
+]
+checksums = [
+    'ee7e5633382bcf9866852e6eaa7bae9000ffb92d6f35605b0031b8d836c4cd94',  # v0.3.1.tar.gz
+    'c470bccdef90b300e8b50d27e7850cc5789ecd1045a85bd8625088c7c7e47a36',  # gloo-20180105.tar.gz
+    '175a22300b3450e27e5f2e6f95cc9abca74617cbc21a1e0ed19bdfbd22ea0305',  # googletest-20160714.tar.gz
+    '7d18b9bdc10de382da1643c2b1b0227131db47eef89998050b5e0a829d3bd0af',  # nanopb-20170723.tar.gz
+    '3a53dac15e87056a9724b9b6403343b47bb71fec26c090979562f71d7ec9ee98',  # pybind11-20170821.tar.gz
+    '828e0d6238e2129a9e08071750dc16ba10e38eacf96f21b8a71e501c2085b282',  # clang-cindex-python3-20170330.tar.gz
+    '2b44b3bcda05acbd58c03a6be5a491ad43f58a31a2d0ba845d241a32df47a136',  # PyTorch-0.3.1-fix-for-intel.patch
+    '710bbd38973a94c5c20695d25b819c4b41565bd4723ee4e07542d9537fa2bf18',  # PyTorch-0.3.1-cmake.patch
+    '72c8d6804bff568c2a49b1b7019171c93309a1853a90d5e3abe778f8aa80ca4f',  # PyTorch-0.3.1_skip_MPI_tests.patch
+    '2c07d9c3622986df3ec8d7f761dc6984f1b44c7a377cad7d7667a030aaeb526f',  # PyTorch-0.3.1_intel_skip_autgrad_test.patch
+    '951d5fde89df3be98c1bc6750546259c1b3d0c77574fa3f9e67d9e8fe5b08161',  # PyTorch-0.3.1_fix_findAVX.patch
+    '400c2e5c2fc4ec0b03d8b95c4612fa174f924e65c23e83b50e21556a36d8f293',  # PyTorch-0.3.1_Fix_pushback_type.patch
+    'b0a6dc564aff1cd6aba0c0c77175413c8f8dc03e771d374755df48365d84bb74',  # PyTorch-0.3.1_Fix_static_assert.patch
+]
+
+# PyTorch pulls in a bunch of submodules which don't have releases.
+# We download the submodule revisions from their repos.
```

Comment:
These comments are probably in the wrong place... used --inject-checksum perhaps?